versions/1/input/manifest.spec.yml

```diff
@@ -0,0 +1,131 @@
+##
+## Describes the specification for the input package's main manifest.yml file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: object
+  additionalProperties: false
+  properties:
+    format_version:
+      description: The version of the package specification format used by this package.
+      $ref: "../integration/manifest.spec.yml#/definitions/version"
+    name:
+      description: The name of the package.
+      type: string
+      pattern: '^[a-z0-9_]+$'
+      examples:
+      - apache
+    title:
+      description: The title of the package.
+      type: string
+      examples:
+      - Apache
+    description:
+      description: A longer description of the package.
+      type: string
+      examples:
+      - Apache Integration
+    version:
+      description: The version of the package.
+      $ref: "../integration/manifest.spec.yml#/definitions/version"
+    license:
+      description: The license under which the package is being released.
+      type: string
+      enum:
+      - basic
+      default: basic
+      examples:
+      - basic
+    type:
+      description: The type of package.
+      type: string
+      enum:
+      - input
+      examples:
+      - input
+    categories:
+      $ref: "../integration/manifest.spec.yml#/definitions/categories"
+    conditions:
+      description: Conditions under which this package can be installed.
+      type: object
+      additionalProperties: false
+      properties:
+        kibana:
+          description: Kibana conditions
+          type: object
+          additionalProperties: false
+          properties:
+            version:
+              type: string
+              description: Kibana versions compatible with this package.
+              examples:
+                - ">=7.9.0"
+    policy_templates:
```

Comment:
Could we `$ref` the whole policy template definition? Or this one has differences with the `integration` one?